docs/github-action-templates.md

```diff
@@ -0,0 +1,68 @@
+# Github Actions templates
+
+This repository contains a number of [GitHub
+Actions](https://docs.github.com/en/actions) workflow template files that
+are copied to a project when running `itkdev-docker-compose
+template:install`. Any changes to the workflows should be made in [this
+repository](%THIS_REPOSITORY%) and then the project template must be updated
+to match the new templates.
+
+---
+
+[github/workflows/changelog.yaml](github/workflows/changelog.yaml)
+
+## Changelog
+
+Checks that changelog has been updated
+
+---
+
+[github/workflows/composer.yaml](github/workflows/composer.yaml)
+
+## Composer
+
+Validates composer.json and checks that it's normalized.
+
+### Assumptions
+
+1. A docker compose service named `phpfpm` can be run and `composer` can be
+   run inside the `phpfpm` service.
+2. [ergebnis/composer-normalize](https://github.com/ergebnis/composer-normalize)
+   is a dev requirement in `composer.json`:
+
+   ``` shell
+   docker compose run --rm phpfpm composer require --dev ergebnis/composer-normalize
+   ```
+
+   Normalize `composer.json` by running
+
+   ``` shell
+   docker compose run --rm phpfpm composer normalize
+   ```
```

Comment:
Is this indented because it is inside a list?

Reply:
Yes.